cmd/orch-tester/orch_tester.go

```diff
@@ -184,6 +225,15 @@ func main() {
 			glog.Error(err)
 			continue
 		}
+		// if we haven't found a random sample by now cancel and wait for the backup attempt to complete before returning
+		if *randomSample {
+			cancel()
+			<-randErr
+			if err != nil {
```

Comment:
This handles the err var defined outside of this conditional, but we want to handle the error returned from the `randErr` channel.

Reply:
fixed in e0ee351